AlttpRandomizer/Random/Randomizer.cs

```diff
@@ -75,7 +75,8 @@ public string CreateRom(RandomizerOptions options)
             {
                 if (options.Filename.Contains("\\") && !Directory.Exists(options.Filename.Substring(0, options.Filename.LastIndexOf('\\'))))
                 {
-                    Directory.CreateDirectory(options.Filename.Substring(0, options.Filename.LastIndexOf('\\')));
+                    String filePath = FileName.Fix(options.Filename, string.Format(romLocations.SeedFileString, seed), complexity);

```

Comment:
Of course you are correct, I have moved the creation of the filePath string to occur just before the if statement and updated the if statement to use that instead of the original path.